spec/requests/apartments_spec.rb

```diff
@@ -1,7 +1,32 @@
 require 'rails_helper'
 
 RSpec.describe "Apartments", type: :request do
+  let(:user) { User.create(
+    email: 'test@example.com',
+    password: 'password',
+    password_confirmation: 'password'
+    )
+  }
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+   it 'gets a list of apartments' do
+    apartment = user.apartments.create(
+      street: "456 Bogus Blvd", 
+      unit: "30", 
+      city: "New York",
+      state: "NY",
+      square_footage: 2000, 
+      price: 13000, 
+      bedrooms: 5, 
+      bathrooms: 3, 
+      pets: "Puppets only",
+      image: "https://upload.wikimedia.org/wikipedia/commons/0/00/Sesame_Street_buildings_%28193090534%29.jpg" 
+    )
+    get '/apartments'
+
+    apartment = JSON.parse(response.body)
+    expect(response).to have_http_status(200)
+    expect(apartment.first['street']).to eq("456 Bogus Blvd")
```

Comment:
Let's add some additional tests that ensures that all attributes have the correct content.